spring-data-cassandra/src/main/java/org/springframework/data/cassandra/repository/support/CassandraRepositoryFactory.java

```diff
@@ -70,7 +71,7 @@ public CassandraRepositoryFactory(CassandraOperations cassandraOperations) {
 
 	@Override
 	@SuppressWarnings({ "rawtypes", "unchecked" })
-	protected Object getTargetRepository(RepositoryMetadata metadata) {
+	protected Object getTargetRepository(RepositoryInformation metadata) {
```

Comment:
Hi, this was a result of simplified repository customizations - I created the more narrow focussed PR #35 to fix this.

Cheers,
Thomas